analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/RoslynSymbolicExecution.cs

```diff
@@ -141,19 +138,24 @@ context.Operation.Instance.Kind switch
                 _ => context.State
             };
 
-        private SymbolicContext InvokeChecks(SymbolicContext context, Func<SymbolicCheck, Func<SymbolicContext, ProgramState>> checkDelegate)
+        private void InvokeExitReached(ExplodedNode node)
         {
+            var context = new SymbolicContext(symbolicValueCounter, null, node.State);
+            foreach (var check in checks)
+            {
+                check.ExitReached(context);
```

Comment:
Why does `ExitReached` have a `ProgramState` return value if it is never used?

Reply:
Historical reasons to reuse `InvokeChecks`... I forgot to remove it as it's pointless now

Reply:
Will you remove it in this PR?

Reply:
I did, but I forgot to push the commit 🤦